app/models/calendar.rb

```diff
@@ -70,7 +65,11 @@ def holiday_approaching?(today: Time.zone.today)
     next_after_today.present? && (next_after_today.start_date - today <= 7)
   end
 
-  def academic_year_for(date)
-    calendar_area.academic_year_for(date)
+  private
+
+  def reject_calendar_events(attributes)
+    end_date_date = Date.parse(attributes[:end_date])
+    end_date_default = end_date_date.month == 8 && end_date_date.day == 31
```

Comment:
I've not revisited this at this point!

Reply:
Hah, we should leave it in as a weird artefact...